SimTracker/SiPhase2Digitizer/test/TBeamTest.cc

```diff
@@ -16,14 +16,14 @@
 // system include files
 #include <memory>
 #include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/Framework/interface/MakerMacros.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/ServiceRegistry/interface/Service.h"
 #include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ESWatcher.h"
 #include "FWCore/Utilities/interface/InputTag.h"
-
+#include "DQMServices/Core/interface/DQMEDAnalyzer.h"
```

Comment:
yes, comes up each PR for this package... Lets get these moved to a proper place..